WcaOnRails/config/locales/de.yml

```diff
@@ -3080,6 +3250,18 @@ de:
       round: Runde
       #original_hash: 709a232
       name: Name
+      rank:
+        #original_hash: d4840ff
+        national: National Rank
```

Comment:
Maybe add a note in brackets and actually translate these terms? Something along the lines of "Nationale Ranglistenplatzierung (engl. National Record)"

Reply:
Those were some examples where I was not sure what to do, as the German translation is quite long, but the English original is well-known. Would add your suggestion (but use "engl. National Rank) to stay in sync with the original, right?

Reply:
Arguably you could leave it at the English term because we use it in German colloquial speech as well. But the point of these translations is to be shown as a tooltip when people hover over the "NR" / "CR" / "WR" abbreviations on a person's WCA ID results page. And I'd argue that someone who uses the "Anglizismus" in their daily vocabulary already knows what "CR" means and doesn't need the tooltip. Therefor, people who do need the tooltip likely won't be familiar with the English terms. That's why I suggested the longer explanations in the first place. Does that make sense?

Reply:
Yes, that makes sense indeed - if it's for a tooltip, then I agree to use a comprehensive translation along with the full explanation of the acronym. I always had in mind that there might be size limitations, but in that case it can basically be as long as it needs to be.

Reply:
Well yes, although you should still try to keep it as short as possible within the boundaries of a thorough explanation